app.rb

```diff
@@ -0,0 +1,113 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize(menu)
+    @menu = menu
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No book found!'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+    @menu.show_menu
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No person found!'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    end
+    @menu.show_menu
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    print 'Classroom: '
+    classroom = gets.chomp
+    @people.push(Student.new(classroom, age, name: name, parent_permission: parent_permission))
+    puts
+    puts 'Person created successfuly'
+  end
```

Comment:
- Hey, great job so far! However, there's an error when trying to create a student. Also, you added the classroom while filling the contents for the student, as this, One doesn't know the datatypes if it's a string or a number to be input for the classroom value. Furthermore, there's still an error thrown for the student. See the image below to fix this issue:


![Screenshot from 2023-02-04 11-05-59](https://user-images.githubusercontent.com/59397159/216761305-a0831b08-90f6-44af-9a5a-c358b4a705fa.png)